src/applications/financial-status-report/tests/e2e/fixtures/mocks/mockUser.json

```diff
@@ -57,7 +57,7 @@
           }
         ],
         "vaPatient": true,
-        "mhvAccountState": "NONE"
+        "mhvAccountState": "OK"
```

Comment:
It looks as if most of the changes are in the `src/applications/health-care-supply-reordering` folder. I think that this change should be excluded from this PR.